.dockerignore

```diff
@@ -0,0 +1,2 @@
+dub/src/
+ldc/src/
```

Comment:
I might be wrong but, folder may have ldc version name in it?

Reply:
Not the top level one. You're thinking about `ldc/src/ldc-1.28.0/`. But the whole src for is a build dir